system/security/CMakeLists.txt

```diff
@@ -28,6 +28,6 @@ IF (USE_APR)
   HPCC_ADD_SUBDIRECTORY (plugins/htpasswdSecurity)
 ENDIF(USE_APR)
 HPCC_ADD_SUBDIRECTORY (plugins/singleuserSecurity)
-IF (USE_JWT)
+IF (NOT WIN32)
```

Comment:
@richardkchapman the jwtCache.cpp file has some Linux specific headers for file name / path matching, I don't think there is any point in coaxing a Windows build for this (I added the USE_JWT on a previous PR)?